src/workerd/io/worker-interface.c++

```diff
@@ -198,6 +209,11 @@ kj::Promise<void> RevocableWorkerInterface::request(
       .attach(kj::mv(wrappedResponse));
 }
 
+kj::Promise<void> RevocableWorkerInterface::connect(kj::StringPtr host, const kj::HttpHeaders& headers,
+    kj::AsyncIoStream& connection, ConnectResponse& response) {
+  return worker.connect(host, headers, connection, response);
```

Comment:
We probably want `RevocableWorkerInterface` to cancel byte stream connections the same way it does WebSockets. This is used in particular to disconnect WebSockets from durable objects when the object's code is upgraded -- you'd want the same for a byte stream connection.

Of course, this doesn't really matter until we support byte stream ingress -- you won't be able to `connect()` to a Durable Object until then. But I'm worried if we implement it to pass through for now, we'll forget about it when it starts mattering. So I would KJ_UNIMPLEMENTED this for now with a TODO saying it should treat connections like WebSockets.

```
KJ_UNIMPLEMENTED("TODO(someday): RevocableWorkerInterface::connect() should be implemented to disconnect long-lived connections similar to how it treats WebSockets");
```